pyvista/plotting/mapper.py

```diff
@@ -117,8 +117,8 @@ def scalar_range(self) -> tuple[float, float]:  # numpydoc ignore=RT01
         return self.GetScalarRange()  # type: ignore[attr-defined]
 
     @scalar_range.setter
-    def scalar_range(self, clim):  # numpydoc ignore=GL08
-        self.SetScalarRange(*clim)
+    def scalar_range(self, clim) -> None:  # numpydoc ignore=GL08
+        self.SetScalarRange(*clim)  # type: ignore[attr-defined]
```

Comment:
This ignore is unrelated to `None` annotations. It's because `_BaseMapper` doesn't inherit any vtk methods